lib/db/schema/patch-022-023.sql

```diff
@@ -0,0 +1,263 @@
+-- Add table for storing unverified sessionTokens
+-- and keyFetchTokens.
+CREATE TABLE `unverifiedTokens` (
+  tokenVerificationId BINARY(32) NOT NULL PRIMARY KEY,
+  tokenId BINARY(32) NOT NULL,
+  INDEX unverifiedTokens_tokenId (tokenId)
+) ENGINE=InnoDB;
```

Comment:
In general use, we're going to by querying this table frequently by `tokenId`, and only ever once by `tokenVerificationId`.  So I wonder if it's worth making `tokenId` the primary key and having a separate index on `tokenVerificationId` to make the common case slightly faster.